apps/google-analytics-4/frontend/src/hooks/useSidebarSlug/useSidebarSlug.tsx

```diff
@@ -7,7 +7,7 @@ import useGetFieldValue from '../useGetFieldValue';
 
 const SLUG_FIELD_INPUT_DELAY = 500;
 
-export const useSidebarSlug = (slugFieldInfo: ContentTypeValue) => {
+export const useSidebarSlug = (slugFieldInfo: ContentTypeValue, useTrailingSlash: boolean) => {
   const sdk = useSDK<SidebarExtensionSDK>();
```

Comment:
Instead of passing down useTrailingSlash down to components/into here, what do you think about getting it from the sdk in this hook?

Reply:
Yes, good call!